sysl2/sysl_parser.go

```diff
@@ -137,5 +137,5 @@ func parse(g *sysl.Grammar, input []int, val interface{}) (bool, []int, []interf
 func checkGrammar(g *sysl.Grammar, tokens []int, start string) bool {
```

Comment:
Rename to `grammarIsValid` or some similar predicate form — anything that makes grammatical sense (excuse the pun) in an `if` statement.

Reply:
its actually going to be renamed later to parseGrammar() as I plan to return ast as well.

Reply:
renamed to parseGrammar()